openedx/tests/xblock_integration/xblock_testcase.py

```diff
@@ -333,16 +352,22 @@ def setUpClass(cls):
         # So, skip the test class here if we are not in the LMS.
         if settings.ROOT_URLCONF != 'lms.urls':
             raise unittest.SkipTest('Test only valid in lms')
-
         super(XBlockTestCase, cls).setUpClass()
 
+    def setUp(self):
+        """
+        Call setups of all parents
+        """
+        super(XBlockTestCase, self).setUp()
+
     def get_handler_url(self, handler, xblock_name=None):
         """
         Get url for the specified xblock handler
         """
         return reverse('xblock_handler', kwargs={
             'course_id': unicode(self.course.id),
-            'usage_id': unicode(self.course.id.make_usage_key('done', xblock_name)),
+            'usage_id': unicode(self.course.id.make_usage_key('done',
```

Comment:
Nit: I prefer the style of keeping line continuation to the left as shown below. But as long as this passes pep8/pylint, I'm OK with it (are we running pep8/pylint on files in openedx?).

````
unicode(
    self.course.id.make_usage_key('done', xbock_name)
)